dev/2018/spring/knights-to-kings/knights-to-kings.html

```diff
@@ -0,0 +1,196 @@
+<article class="story story-default">
+  <div class="container">
+    <div class="row">
+        <img src="[image filename='Knights-to-Kings-UCF-Football']" alt="From Knights to Kings" title="From Knights to Kings" class="img-responsive center-block story-header">
+        <h1 class="sr-only">From Knights to Kings: UCF Football players reflect on the 2017 season</h1>
+    </div>
+  </div>
+  <div class="story-deck" [background-image filename="Gold-BG"]>
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-10 col-xs-offset-1 text-center">
+          <h2>Inside the team that went from<br class="hidden-sm hidden-xs"> zero wins to National Champions.</h2>
```

Comment:
Can we change this from an h2 to a non-heading. This isn't really sectional, it's acting more like a subtitle. Also, can we use one of the sans-serif fonts on this. The Oswald would be nice.